.github/ISSUE_TEMPLATE/subdomain-request.md

```diff
@@ -8,16 +8,32 @@ assignees: ''
 ---
 
 **Project Info**
-[Source URL if any](https://example.github.io/)
-[Link to repository URL](https://github.com/example/test)
+
+Source URL if any: 
+Link to repository URL: 
+
+```
+# example
+Source URL if any: https://example.github.io/
```

Comment:
Does this generate a clickable link for GFM

Reply:
https://example.github.io/

^ will automatically become a link

however, I did realise there's a minor formatting snafu, need to add one more line break

Reply:
Ok, commit 9570ff4 fixes that